### ISSUE TITLE
User unable to create a group

### ISSUE THREAD

**Issue by babaklar** [MESH-Research/commons#448]
A [user](https://registry.hcommons.org/registry/co_people/canvas/46471) has reported he's unable to create a Humanities Commons group. When he views the group page, there isn't a "Create a Group" button, even when he logs in. Screenshot is attached. ![Screen Shot 2022-08-29 at 9.30.04 AM.png](https://images.zenhubusercontent.com/62572e0e3e1b973699e0382a/06fa7137-229c-43ff-bf9d-bc7a15668141)

**Comment by mikethicke** [comment]
The user's account had not been provisioned from comanage to the new membership LDAP:

![Screen Shot 2022-08-31 at 7.27.55 AM.png](https://images.zenhubusercontent.com/6108aeed64d7503f6ae51eff/cd8cae8c-5ab4-47b4-8379-3acb8cd11957)

This resulted in them not getting HC membership even when they re-logged in.

Fixes:

1) I provisioned this user, so they should have the correct membership data passed on login.
2) I added fallback organizational membership checking based on the legacy 'people' memberships. This should prevent other users from having the same issue. (https://github.com/MESH-Research/humanities-commons/commit/70f61e1fac2c23989090ab8ca69887350713c89e).

Will do a bulk re-provision this evening to try and catch any other users in a similar state.

@babaklar : If this user logs-out and logs back in, their issue should be corrected.
